pumps/mongo_selective.go

```diff
@@ -199,20 +183,14 @@ func (m *MongoSelectivePump) WriteData(ctx context.Context, data []interface{})
 
 				indexCreateErr := m.ensureIndexes(analyticsCollection)
 				if indexCreateErr != nil {
-					log.WithFields(logrus.Fields{
-						"prefix": mongoSelectivePrefix,
-					}).Error(indexCreateErr)
+					m.log.Error(indexCreateErr)
```

Comment:
@tbuchaillot since we already enhance the loggings, can we add hear more context? for example, the name of the collection

Reply:
Done!